yaml/minimal.yaml

```diff
@@ -4,15 +4,8 @@ strategy:
   sequence: # example for a batch sequence
     foo: [0.1, 0.2, 0.3, 0.4, 0.5, 0.6, 0.7, 0.8]
   matrix: # example for a batch matrix
-    foo:
-      limits: [0.1, 0.3]
-      npoints: 3
-      mode: linspace
-    bar:
-      limits: [3, 0]
-      step: 1
-      mode: arange
-      reverse: True
+    foo: { mode: linspace , limits: [0.1, 0.3], npoints: 3 }
+    bar: { mode: arange, limits: [3, 0], step: 1, reverse: True }
```

Comment:
@DavidAkinpelu ... I reformatted this for brevity (which doesn't change a thing), but have a short question before I merge. Any reason to not specify `step: -1` as an alternative?

Reply:
It's not necessary. I will remove it.